integrationtest/backend/lucene/src/test/java/org/hibernate/search/integrationtest/backend/lucene/fieldtype/LuceneFieldContentIT.java

```diff
@@ -0,0 +1,195 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.backend.lucene.fieldtype;
+
+
+import static org.hibernate.search.integrationtest.backend.lucene.LuceneExtensionIT.containsDocument;
+import static org.hibernate.search.util.impl.integrationtest.common.stub.mapper.StubMapperUtils.referenceProvider;
+
+import java.util.Arrays;
+import java.util.List;
+
+import org.hibernate.search.backend.lucene.LuceneExtension;
+import org.hibernate.search.engine.backend.document.DocumentElement;
+import org.hibernate.search.engine.backend.document.IndexFieldAccessor;
+import org.hibernate.search.engine.backend.document.model.dsl.IndexSchemaElement;
+import org.hibernate.search.engine.backend.index.spi.IndexWorkPlan;
+import org.hibernate.search.engine.backend.types.Projectable;
+import org.hibernate.search.engine.search.SearchQuery;
+import org.hibernate.search.integrationtest.backend.tck.testsupport.configuration.DefaultAnalysisDefinitions;
+import org.hibernate.search.integrationtest.backend.tck.testsupport.util.rule.SearchSetupHelper;
+import org.hibernate.search.util.impl.integrationtest.common.stub.mapper.StubMappingIndexManager;
+import org.hibernate.search.util.impl.integrationtest.common.stub.mapper.StubMappingSearchTarget;
+import org.hibernate.search.util.impl.test.annotation.TestForIssue;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+
+import org.apache.lucene.document.Document;
+import org.apache.lucene.document.IntPoint;
+import org.apache.lucene.document.NumericDocValuesField;
+import org.apache.lucene.document.StoredField;
+import org.assertj.core.api.Assertions;
+
+@TestForIssue(jiraKey = "HSEARCH-1640")
+public class LuceneFieldContentIT {
+
+	private static final String INDEX_NAME = "IndexName";
+	private static final String TEXT_1 = "This is a text containing things. Red house with a blue carpet on the road...";
+	private static final String TEXT_2 = "This is a text containing other things. Such as move the line on the right margin...";
+
+	@Rule
+	public SearchSetupHelper setupHelper = new SearchSetupHelper();
+
+	private IndexAccessors indexAccessors;
+	private StubMappingIndexManager indexManager;
+
+	@Before
+	public void setup() {
+		setupHelper.withDefaultConfiguration( "myLuceneBackend" )
+				.withIndex(
+						"MappedType", INDEX_NAME,
+						ctx -> this.indexAccessors = new IndexAccessors( ctx.getSchemaElement() ),
+						indexManager -> this.indexManager = indexManager
+				)
+				.setup();
+
+		initData();
+	}
+
+	@Test
+	public void verifyProjectionsOnDifferentTypes_usingStandardAPI() {
```

Comment:
This is already tested extensively in the backend TCK. You can remove this test, it's not worth the maintenance cost.